### PR TITLE
Address #77 and #79

### PR DESCRIPTION
**Pull request by dubois**

### CODE REVIEW

File: Support/Python/tbdata/brush_lookup.py [hunk 1]
```diff
@@ -11,51 +11,67 @@
...
+    with open(os.path.join(self.dir, 'Assets/Manifest.asset')) as inf:
+      data = inf.read().replace('\r', '')
+    brush_chunk = re.search(r'Brushes:\n(  -.*\n)*', data).group(0)
+    for match in re.finditer(r'guid: ([0-9a-f]{32})', brush_chunk):
+      yield cast(Guid, match.group(1))
+
   @staticmethod
-  def iter_brush_guid_and_name(tilt_brush_dir):
+  def iter_brush_guid_and_name(tilt_brush_dir: str) -> Iterator[Tuple[Guid, str]]:
+    """Yields (guid, name, is_experimental) tuples."""
```
Comment (TimAidley) [inline Support/Python/tbdata/brush_lookup.py:37]: This docstring looks incorrect - where does is_experimental get returned?

**Review by TimAidley — APPROVED** [review]
Only found one minor nit. LGTM.

File: Assets/Editor/Tests/TestMathUtils.cs [hunk 2]
```diff
@@ -810,5 +810,15 @@ internal class TestMathUtils : MathTestUtils {
     Assert.AreEqual(floats[0], floats1000[0]);
     Assert.AreEqual(floats[floats.Length-1], floats1000[floats1000.Length-1]);
   }
+
+  [Test]
```
Comment (mikeage) [inline Assets/Editor/Tests/TestMathUtils.cs:814]: Awesome! It doesn't look like all tests are functional right now, so we're not running them in CI, but I love that people are adding tests :-)

File: Support/Python/tbdata/brush_lookup.py [hunk 3]
```diff
@@ -11,51 +11,67 @@
...
           if f.lower().endswith('.asset'):
             fullf = os.path.join(r, f)
-            with open(fullf) as f:
-              data = f.read()
-            guid = re.search('m_storage: (.*)$', data, re.M).group(1)
+            with open(fullf) as inf:
+              data = inf.read()
+            guid = cast(Guid, re.search('m_storage: (.*)$', data, re.M).group(1))
             # name = re.search('m_Name: (.*)$', data, re.M).group(1)
+            name = f[:-6]
```
Comment (mikeage) [inline Support/Python/tbdata/brush_lookup.py:47]: that conflation of `f` above and `f` here was probably my fault, from #35. Thanks!

File: Support/Python/tbdata/brush_lookup.py (hunk 1 above)
Comment (dubois) [reply Support/Python/tbdata/brush_lookup.py:37]: Done.
Oops! That got added then removed.

**Comment by dubois** [comment]
Nit fixed.

**Pull request merged**